### PR TITLE
feat(client-sdks): add batch evaluation to engine

### PR DESCRIPTION
**Pull request by yquansah**
Add batch evaluation to engine + clients.

This was missing previously, so we are adding it for users to be able to leverage the batch evaluation functionality.

### CODE REVIEW

**Comment by yquansah** [comment]
Just got to do ruby. I'll attack that tomorrow lol

File: flipt-client-go/models.go [hunk 1]
```diff
@@ -7,6 +7,12 @@ type evaluationRequest struct {
 	Context      map[string]string `json:"context"`
 }
 
+type InputEvaluationRequest struct {
```
Comment (markphelps) [inline flipt-client-go/models.go:10]:
```suggestion
type EvaluationRequest struct {
```

Comment (markphelps) [reply flipt-client-go/models.go:10]: wdyt about just EvaluationRequest for all these?

Comment (yquansah) [reply flipt-client-go/models.go]: @markphelps Yeah I didnt want to expose the fact that you can input a namespace. But there is a cleaner way to do this. Pushed up another commit.

**Review by markphelps — APPROVED** [review]
one minor change, otherwise lgtm!! thanks

**Pull request merged**